### DISCUSSION TITLE
Vite build flags

### DISCUSSION THREAD

**Discussion by BillBuilt** [sveltejs/kit#3478] [Q&A]
Hello,
Vite's build command has a`--watch` flag that `rebuilds when modules have changed on disk`. I've been experimenting using Svelte in Wails ([wails.io](https://wails.io/)) and this is necessary to work alongside of the Go live-reloading feature. I just change the npm run command to `vite build --watch`. I was hoping to play around with SvelteKit in Wails, but I can't figure out how to achieve this in SvelteKit. Is, or would, it possible to do this or something similar using SvelteKit too? 

Thanks

**Comment by mustofa-id** [comment]
why don't use vite [config ](https://vitejs.dev/config/#build-watch) instead?

```js
// svelte.config.js file
/** @type {import('@sveltejs/kit').Config} */
const config = {
	// ...
	kit: {
		// ...
		vite: {
			build: {
				watch: {} // here
			},
		}
	}
}
```

**Comment by BillBuilt** [comment]
Hello,

Thanks for the suggestion. Unfortunately, this results in the following error when issuing `npm run build`:

```
> svelte-kit build

vite v2.7.13 building for production...

watching for file changes...
> Cannot read properties of undefined (reading 'filter')
    at create_build (file:///projects/test/node_modules/@sveltejs/kit/dist/chunks/index3.js:102:10)
    at async build_client (file:///projects/test/node_modules/@sveltejs/kit/dist/chunks/index3.js:220:29)
    at async build (file:///projects/test/node_modules/@sveltejs/kit/dist/chunks/index3.js:615:17)
    at async file:///Users/billmatlock/web/htdocs/Golang/ledgerapp.kit/frontend/node_modules/@sveltejs/kit/dist/cli.js:986:23

```
